features/user_can_log_in.feature

```diff
@@ -0,0 +1,26 @@
+Feature: User can log in
+  As a User 
+  In order to use the application
+  I would like to be able to log in to my account 
+
+  //Background: 
+   // Given I visit the "landing" page
+    //And I click "Login" button 
```

Comment:
Please remove comments before you submit PR for review. Also is "//" actually commenting out in Gherkin"?

Reply:
I should have used # .